### ISSUE TITLE
every new tag for every enter

### ISSUE THREAD

**Issue by Amara-Balakrishna1** [NYTimes/ice#121]
![capture](https://cloud.githubusercontent.com/assets/9972828/9975654/2577a366-5ee1-11e5-8af6-ebf3aa269ee3.PNG)
![capture1](https://cloud.githubusercontent.com/assets/9972828/9975655/2ba54018-5ee1-11e5-8f2c-868190f9dac4.PNG)
it should look like second image is it possible

**Comment by Amara-Balakrishna1** [comment]
Adding table is not  tracked

**Comment by catsgotmytongue** [comment]
Yes it is possible, look at the options for tinymce on itself and ICE. It's
been a while since I worked on it. I was considering starting a new project
to be a tinymce replacement written in typescript and hosted on github,
with built in change tracking but frankly I have not had much time lately
between work and home life.

However, you should definitely look into the options as there are lots of
them. I think the option you want has to do with root element or p options.

**Comment by Amara-Balakrishna1** [comment]
there are no options for the removing of p tag.im doing hte process of redlining like google docs doing.suggetion option in google docs.please lookhttps://docs.google.com/document/u/0/?pli=1
